vnext/src/Libraries/Components/AccessibilityInfo/AccessibilityInfo.windows.js

```diff
@@ -39,147 +65,286 @@ const _subscriptions = new Map();
+  addEventListener<K: $Keys<AccessibilityEventDefinitions>>(
     eventName: K,
     handler: (...$ElementType<AccessibilityEventDefinitions, K>) => void,
   ): EventSubscription {
-    let listener;
-
-    if (eventName === 'change' || eventName === 'screenReaderChanged') {
-      listener = RCTDeviceEventEmitter.addListener(
-        TOUCH_EXPLORATION_EVENT,
-        handler,
-      );
-    } else if (eventName === 'reduceMotionChanged') {
-      listener = RCTDeviceEventEmitter.addListener(
-        REDUCE_MOTION_EVENT,
-        handler,
-      );
-    }
-
-    // $FlowFixMe[escaped-generic]
-    _subscriptions.set(handler, listener);
-
-    return {
-      remove: () => {
-        // $FlowIssue flow does not recognize handler properly
-        AccessibilityInfo.removeEventListener<K>(eventName, handler);
-      },
-    };
-  },
-
-  removeEventListener: function<K: $Keys<AccessibilityEventDefinitions>>(
-    eventName: K,
-    handler: (...$ElementType<AccessibilityEventDefinitions, K>) => void,
-  ): void {
-    // $FlowFixMe[escaped-generic]
-    const listener = _subscriptions.get(handler);
-    if (!listener) {
-      return;
-    }
-    listener.remove();
-    // $FlowFixMe[escaped-generic]
-    _subscriptions.delete(handler);
+    const deviceEventName = EventNames.get(eventName);
+    return deviceEventName == null
+      ? {remove(): void {}}
+      : RCTDeviceEventEmitter.addListener(deviceEventName, handler);
   },
 
   /**
-   * Set accessibility focus to a react component.
+   * Set accessibility focus to a React component.
    *
    * See https://reactnative.dev/docs/accessibilityinfo.html#setaccessibilityfocus
    */
-  setAccessibilityFocus: function(reactTag: number): void {
+  setAccessibilityFocus(reactTag: number): void {
     legacySendAccessibilityEvent(reactTag, 'focus');
   },
 
   /**
    * Send a named accessibility event to a HostComponent.
    */
-  sendAccessibilityEvent_unstable: function(
+  sendAccessibilityEvent_unstable(
     handle: ElementRef<HostComponent<mixed>>,
     eventType: AccessibilityEventTypes,
   ) {
-    // route through React renderer to distinguish between Fabric and non-Fabric handles
     // iOS only supports 'focus' event types
-    if (eventType === 'focus') {
-      sendAccessibilityEvent(handle, eventType);
-    } else if (eventType === 'click') {
-      // Do nothing!
+    if (Platform.OS === 'ios' && eventType === 'click') {
+      return;
     }
+    // route through React renderer to distinguish between Fabric and non-Fabric handles
+    sendAccessibilityEvent(handle, eventType);
   },
 
   /**
    * Post a string to be announced by the screen reader.
    *
    * See https://reactnative.dev/docs/accessibilityinfo.html#announceforaccessibility
    */
-  announceForAccessibility: function(announcement: string): void {
-    if (NativeAccessibilityInfo) {
-      NativeAccessibilityInfo.announceForAccessibility(announcement);
```

Comment:
Example of what I said above - our implementation of `NativeAccessibilityInfo.announceForAccessibility` will get skipped now